config/locales/en/prison_views.yml

```diff
@@ -229,6 +228,7 @@ en:
         issue_with_date: Issue with the date
         visitor_banned: Visitor is banned
         issue_with_visitor: Issue with a visitor
+        cancel_to_nomis_warning: This booking will automatically be cancelled in NOMIS
```

Comment:
@kayesoutham could you provide me with some adequate content to replace this with? 🙏  🍰  🎃

Reply:
@StupidCodeFactory Should we be giving the user the chance to skip this as we do when processing the visit? A checkbox to "Don't automatically cancel this visit to NOMIS"?

Reply:
yes you're correct, I hadn't open the image in trello :/

Reply:
Also, this text is hardcoded, should be dependent on the feature being enabled.